core/src/main/web/app/beaker.js

```diff
@@ -272,18 +272,21 @@
       }
     });
 
-    beaker.run(function($location, $route, $document, bkUtils, bkCoreManager, bkHelper) {
+    beaker.run(function($rootScope, $location, $route, $document, bkUtils, bkCoreManager, bkHelper) {
       var user;
       var lastAction = new Date();
       var beakerRootOp = {
         gotoControlPanel: function() {
-          return $location.path("/control").search({});
+          var ret = $location.path("/control").search({});
+          if (bkUtils.isElectron) {
+            $rootScope.$apply();
```

Comment:
For some reason simply changing $location has no effect when Beaker is running on Electron, the apply is needed to make angular update. I don't think there is a way to call 'gotoControlPanel' right now from Electron (there was in the infancy of my branch), but it is probably a good idea to keep this here in case we ever implement redirection to the control panel?